google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts
+Before=shutdown.target reboot.target halt.target
+DefaultDependencies=true
+Wants=local-fs.target network-online.target network.target
```

Comment:
Fixed the default dependencies. Wants is needed because shutdown scripts using network shouldn't have network cut out early.